app/components/candidate_interface/gcse_equivalency_required_component.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if !provider_accepts_equivalencies? && @missing_uk_gcses.any? %>
```

Comment:
So there's one thing that we've not considered here that @frankieroberto might want to give input in. Providers have the ability to accept equivalencies in some subjects and not others. 

So here i've set it up locally that they accept equivs for maths, but not English. 


![image](https://user-images.githubusercontent.com/42515961/133780850-0d9fe96f-35e2-4097-b7d4-987393ef022b.png)

And because we're just checking the `application_choice.course.accept_gcse_equivalency?` boolean, it doesn't realise that they have a missing qualification in English (which they don't accept equivalencies for)

![image](https://user-images.githubusercontent.com/42515961/133781081-fce831af-b333-4ded-add1-b60f607cc304.png)
.